cty/value.go

```diff
@@ -107,10 +107,10 @@ func (val Value) IsWhollyKnown() bool {
 	}
 }
 
-// HasDynamicValues checks if the value is dynamic, or contains any nested
+// HasWhollyKnownType checks if the value is dynamic, or contains any nested
 // DynamicVal. This implies that both the value is not known, and the final
 // type may change.
-func (val Value) HasDynamicValues() bool {
+func (val Value) HasWhollyKnownType() bool {
```

Comment:
It looks like you inverted the direction of the name here but not the implementation along with it... if I'm reading correctly, the logic below seems to produce an answer to "Doesn't have a wholly known type".

Reply:
😆 oh yes, I just did a find and replace 😞 Will fix ASAP